Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -16,276 +16,298 @@
+                              return try m.serializedData(partial: true)
+                        }
+                        catch { return Internal.emptyData }
+                  }
+            }
+            set { state = .binary(newValue) }
       }
-    }
-    set {
-      state = .binary(newValue)
-    }
-  }
-
-  enum InternalState {
-    // a serialized binary
-    // Note: Unlike contentJSON below, binary does not bother to capture the
-    // decoding options. This is because the actual binary format is the binary
-    // blob, i.e. - when decoding from binary, the spec doesn't include decoding
-    // the binary blob, it is pass through. Instead there is a public api for
-    // unpacking that takes new options when a developer decides to decode it.
-    case binary(Data)
-    // a message
-    case message(Message)
-    // parsed JSON with the @type removed and the decoding options.
-    case contentJSON(Data, JSONDecodingOptions)
-  }
-  var state: InternalState = .binary(Internal.emptyData)
-
-  static let defaultInstance = AnyMessageStorage()
-
-  private init() {}
-
-  init(copying source: AnyMessageStorage) {
-    _typeURL = source._typeURL
-    state = source.state
-  }
-
-  func isA<M: Message>(_ type: M.Type) -> Bool {
-    if _typeURL.isEmpty {
-      return false
-    }
-    let encodedType = typeName(fromURL: _typeURL)
-    return encodedType == M.protoMessageName
-  }
-
-  // This is only ever called with the expactation that target will be fully
-  // replaced during the unpacking and never as a merge.
-  func unpackTo<M: Message>(
-    target: inout M,
-    extensions: ExtensionMap?,
-    options: BinaryDecodingOptions
-  ) throws {
-    guard isA(M.self) else {
-      throw AnyUnpackError.typeMismatch
-    }
-
-    switch state {
-    case .binary(let data):
-      target = try M(serializedData: data, extensions: extensions, partial: true, options: options)
-
-    case .message(let msg):
-      if let message = msg as? M {
-        // Already right type, copy it over.
-        target = message
-      } else {
-        // Different type, serialize and parse.
-        let data = try msg.serializedData(partial: true)
-        target = try M(serializedData: data, extensions: extensions, partial: true)
+
+      enum InternalState {
+            // a serialized binary
+            // Note: Unlike contentJSON below, binary does not bother to capture the
+            // decoding options. This is because the actual binary format is the binary
+            // blob, i.e. - when decoding from binary, the spec doesn't include decoding
+            // the binary blob, it is pass through. Instead there is a public api for
+            // unpacking that takes new options when a developer decides to decode it.
+            case binary(Data)
+            // a message
+            case message(Message)
+            // parsed JSON with the @type removed and the decoding options.
+            case contentJSON(Data, JSONDecodingOptions)
```

Comment:
Probably, for all tests that are stripping discretionary breaks, the thing to do is make the PR against an empty repo, so all lines are additions and there are no deletions to interrupt our view of the reformatted code.  We don't really care what the old code looked like for this kind of test.